<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBook</class>
 <widget class="QDialog" name="AddBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bookLine">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>201</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="autorLine">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>201</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Название</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Автор</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> Год издания</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Жанр</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="genreLine">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>201</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
